crates/sparrow-expressions/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition.workspace = true
 license.workspace = true
 publish = false
 description = """
-Execution of expressions over record batches.
+Implementation of stateless expressions over batches.
```

Comment:
I forgot if we added machete to CI, but did you run it here? Figured maybe `sparrow-physical` wouldn't be needed anymore here

Reply:
We had added it, but due to the default working directory, it only affected Python. I may change the default back to that.